layouts/partials/ecosystem/registry/entry.html

```diff
@@ -116,16 +116,19 @@ <h4 class="card-title mb-0 d-flex flex-row">
             <span class="badge rounded-pill text-bg-info"><i class="fa-regular fa-star"></i> new</span>
             {{ end -}}
             {{ if $isNative -}}
-            <span class="badge rounded-pill text-bg-success text-white"><i class="fa-solid fa-puzzle-piece"></i> native</span>
+            <a href="/ecosystem/registry?flag=native" class="badge rounded-pill text-bg-success text-white" title="Click to filter by native tag">
```

Comment:
we might change that but we currently call them "flags"

```suggestion
            <a href="/ecosystem/registry?flag=native" class="badge rounded-pill text-bg-success text-white" title="Click to filter by native flag">
```